src/resolve/metadataResolver.ts

```diff
@@ -78,9 +78,11 @@ export class MetadataResolver {
       if (this.tree.isDirectory(fsPath)) {
         if (this.resolveDirectoryAsComponent(fsPath)) {
           const component = this.resolveComponent(fsPath, true);
-          if (!inclusiveFilter || inclusiveFilter.has(component)) {
-            components.push(component);
-            ignore.add(component.xml);
+          if (component) {
+            if (!inclusiveFilter || inclusiveFilter.has(component)) {
```

Comment:
Can we combine the if statements to be ` if (component && (!inclusiveFilter || inclusiveFilter.has(component)))`

Reply:
sure